pkg/nfd-topology-updater/kubeletnotifier/kubeletnotifier.go

```diff
@@ -31,6 +32,8 @@ type EventType string
 const (
 	IntervalBased EventType = "intervalBased"
 	FSUpdate      EventType = "fsUpdate"
+
+	devicePluginsDirName = "device-plugins"
```

Comment:
Maybe it is better to change Kubelet-state-dir to a [slice instead of string](https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/cmd/nfd-topology-updater/main.go#L149) and specifies the device-plugins as part of the configuration. 
Then in the configuration we'll specify
 `--kubelet-state-dir=/var/lib/kubelet --kubelet-state-dir=/var/lib/kubelet/device-plugins`

Reply:
I like the idea of allowing passing more than one dir here but `kubelet_checkpoint_internal` is hardcoded as a default file to watch and we are failing in doing it since we setup watchers for wrong directory. We should either fix what we have right now or allow flexibility in specifying `files` to watch not dirs :)

Reply:
Switching to files sounds good to me but maybe it deserves a separate PR and the current implementation and fix are good enough for this pr